index.html

```diff
@@ -3,47 +3,105 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>Week 1: HTML Tables</title><!-- Latest compiled and minified CSS -->
+  <title>Democratic Delegate Count</title><!-- Latest compiled and minified CSS -->
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
   <style>
+   
+
+
+    tr:hover td {
+        background-color: #f5f5f5 
+    }
+
+    tr:hover .winner{
+      font-weight: bolder;
+    }
+
+    .winner{
```

Comment:
Big 👍 on using `winner` as the class. We'd call this "semantic" markup, in that it describes what the content is, not how it looks. The alternative, using a class like `highlighted` would not be semantic, because it describes the desired styles, not the reason the content is so styled.